kuma/static/styles/components/home/home-masthead.scss

```diff
@@ -20,7 +20,7 @@ homepage masthead (search, feature callouts)
     }
 
     h1 {
-        @include set-font-size(36px);
+        @include set-font-size(42px);
```

Comment:
Shouldn’t this be using `$h2-font-size` or&nbsp;another such variable?

Reply:
Well, yes and no ;) I am working on the side on a more modular scale for our typography with some meaningful class/variable names. It just feels strange to me to set the `font-size` for an `h1` to `$h2-font-size`